frontend/src/context/AppContext.tsx

```diff
@@ -97,12 +89,7 @@ export function AppProvider({ children }: AppProviderProps) {
       setNotes(notes.map((note) => (note.id !== id ? note : returnedNote)));
     } catch (error) {
       if (error instanceof Error) {
-        setErrorMessage(
-          `Note '${note.content}' was already removed from server`
-        );
-        setTimeout(() => {
-          setErrorMessage("");
-        }, 5000);
+        toast.error(`Note '${note.content}' was already removed from server`);
```

Comment:
Print the error message instead. We don't know if note is actually removed or not.